coderedcms/templatetags/coderedcms_tags.py

```diff
@@ -151,3 +133,21 @@ def richtext_amp(value):
 
     value = utils.convert_to_amp(value)
     return mark_safe(value)
+
+@register.filter
+def amp_formatting(value):
+    return mark_safe(utils.convert_to_amp(value))
+
+@register.simple_tag
+def render_iframe_from_embed(embed):
+    soup = BeautifulSoup(embed.html, "html5lib")
+    try:
+        iframe_tags = soup.find('iframe')
+        iframe_tags['title'] = embed.title
+        return mark_safe(str(soup.body.iframe))
```

Comment:
Returning `soup.body.iframe` seems like it might miss something, e.g. what if `embed.html` contains a div wrapper around an iframe; then you would lose the wrapper div, right? Couldn't you just return the whole `soup`, or does that add a bunch of unwanted HTML markup?